notebooks/COS/LSF/LSF.ipynb

```diff
@@ -1805,8 +1635,11 @@
     "tar = tarfile.open(readfile, \"r:gz\")\n",
     "tar.extractall()\n",
     "tar.close()\n",
-    "stis_nuv_cspec = \"./bi173/hlsp_ullyses_hst_stis_bi173_e230m_dr2_cspec.fits\"\n",
-    "\n",
+    "stis_data_dir = Path(\"./bi173/\")\n",
```

Comment:
The only salient change. It falls in the "minor change" category of the COS write access rules.